python/app/shotgun_formatter.py

```diff
@@ -235,10 +236,12 @@ def _sg_field_to_str(self, sg_type, sg_field, value, directive=None):
             if not self._generates_links(value["type"]) or directive == "nolink":
                 str_val = link_name
             else:
-                str_val = "<a href='sgtk:%s:%s' style='text-decoration: none; color: %s'>%s</a>" % (value["type"], 
-                                                                                                    value["id"],
-                                                                                                    self._app.style_constants["SG_HIGHLIGHT_COLOR"], 
-                                                                                                    link_name)
+                str_val = get_hyperlink_html(
+                    "sgtk:%s:%s" % (value["type"], value["id"]),
+                    link_name,
+                    self._app.style_constants["SG_HIGHLIGHT_COLOR"],
```

Comment:
Per discussion, it's now bold foreground color for everything in the panel. I had also talked about removing the bold from some of the list item labels (like "By: " and the like) but I decided against that after a second look.